Lib/importlib/abc.py

```diff
@@ -381,8 +381,8 @@ def is_resource(self, name):
 
     @abc.abstractmethod
     def contents(self):
-        """Return an iterator of strings over the contents of the package."""
-        return iter([])
+        """Return an iterable of strings over the contents of the package."""
+        return []
```

Comment:
Maybe return a tuple? Creating and iterating a tuple is a tiny bit faster, and the empty tuple consumes less memory.

Reply:
+0 - I leave it up to @brettcannon